CHANGELOG.md

```diff
@@ -5,6 +5,8 @@
   on a TableView. Applying this to a TableView does not modify the results, but gives
   users the ability to use the reporting method to find rows in a different table that
   are connected by backlinks. This is intened for sync subscriptions.
+* A new function, Query::has_read_permission() has been added. The function produces a query
```

Comment:
Is it possible express the functionality in Core's own terms? Possibly by generalizing the functionality? This seems like introducing concepts from a specific implementation in Sync into the Core project - likely adding technical debt. Do we want to do it this way?